src/euphorie/client/browser/webhelpers.py

```diff
@@ -165,9 +165,11 @@ def use_involve_phase(self):
     @property
     @memoize
     def use_training_module(self):
-        return api.portal.get_registry_record(
+        globally_enabled = api.portal.get_registry_record(
             "euphorie.use_training_module", default=False
         )
+        country_enabled = self.content_country_obj.enable_web_training
```

Comment:
It is useless to get the `country_enabled` if globally_enabled is False.

Reply:
I'll change that, thx.